gradio/blocks.py

```diff
@@ -797,6 +807,8 @@ def set_event_trigger(
             trigger_after: if set, this event will be triggered after 'trigger_after' function index
             trigger_only_on_success: if True, this event will only be triggered if the previous event was successful (only applies if `trigger_after` is set)
             trigger_mode: If "once" (default for all events except `.change()`) would not allow any submissions while an event is pending. If set to "multiple", unlimited submissions are allowed while pending, and "always_last" (default for `.change()` event) would allow a second submission after the pending event is complete.
+            concurrency_limit: If set, this this is the maximum number of this event that can be running simultaneously. Extra events triggered by this listener will be queued. On Spaces, this is set to 1 by default.
```

Comment:
Can't remember, why are we setting this to be 1 only on Spaces? I think it makes sense to set it to 1 locally (since you could be using a GPU-based function locally too) and better to have a unified experience locally and on Spaces